bentoml/sklearn.py

```diff
@@ -137,31 +147,32 @@ def __init__(
 
```

Comment:
we should also change the parallel_backend mode above, from 'threading' to 'loky': https://scikit-learn.org/stable/modules/generated/sklearn.utils.parallel_backend.html